src/django_idom/components.py

```diff
@@ -164,31 +166,35 @@ def _django_css(static_path: str):
     return html.style(_cached_static_contents(static_path))
 
 
-def django_css(static_path: str):
+def django_css(static_path: str, key: Any | None = None):
```

Comment:
key type is `str | None`

Reply:
I thought keys could be int as well?

Reply:
Ah, you're right. There's actually a type you can just import for this. `idom.types.Key`. So the type would be `Key | None`